usaspending_api/common/views.py

```diff
@@ -81,7 +82,7 @@ class AutocompleteView(AutocompleteResponseMixin,
     def post(self, request, *args, **kwargs):
         try:
             response = self.build_response(
-                self.request, queryset=self.get_queryset(), serializer=self.serializer_class)
+                request, queryset=self.get_queryset(), serializer=self.serializer_class)
             status_code = status.HTTP_200_OK
         except InvalidParameterException as e:
             response = {"message": str(e)}
```

Comment:
Yeah, if you send a checksum that doesn't exist it generates an InvalidParameterException